app-server/app/src/main/scala/user/User.scala

```diff
@@ -2,40 +2,65 @@ package com.azavea.rf.user
 
 import scala.concurrent.{Future, ExecutionContext}
 import scala.util.{Try, Success, Failure}
+import scala.language.postfixOps
 import java.sql.Timestamp
-import slick.lifted.{ColumnOrdered, TableQuery, Query}
+import slick.lifted.Query
 import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
-import de.choffmeister.auth.common.JsonWebToken
 
-import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations}
-import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations, UsersToOrganizations}
+import com.azavea.rf.datamodel.latest.schema.tables.{UsersRow, UsersToOrganizationsRow}
 import com.azavea.rf.utils.{Database,PaginatedResponse}
 
+object Roles extends Enumeration {
```

Comment:
This file is getting kind of  catch-all -- I think we should start splitting some things up.

Maybe roles should go in its own `Roles.scala`

Would be good to try and split this file up further where it makes sense beyond that.